### PR TITLE
Added back button on single track pages

### LINKED ISSUE #1376: Back button on Single track pages
There should be a back button on top left of individual track pages which leads to the main track page.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1376 

Changes: Added back button in session.hbs

Screenshots for the change: 
screenshot from 2017-09-29 18-12-06

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@Princu7 @geekyd please review.

**Comment by geekyd** [comment]
@agbilotia1998 Thanks for the PR, I'm not sure if we need to add a back button here. @mariobehling @Princu7 @aayusharora Can you please confirm?

**Comment by zamhaq** [comment]
@agbilotia1998 @geekyd The back button does not seems to be a requirement since the navigation tab is already present.

**Comment by mariobehling** [comment]
Please check how sched.org is doing it and implement it in the same way. Cheers.

**Comment by agbilotia1998** [comment]
@aayusharora please guide on this.

**Comment by agbilotia1998** [comment]
@mariobehling this PR is awaiting for review from a long time. Please review.Thanks :)

File: src/backend/templates/session.hbs [hunk 1]
```diff
@@ -44,6 +44,9 @@
     {{#session}}
       <div class="container session-container">
         <div class="row single-session" id="{{session_id}}">
+          <a href="javascript:history.back()" style="float:left;">
```
Comment (geekyd) [inline src/backend/templates/session.hbs]: @agbilotia1998 Just remove the inline styling from here

**Comment by agbilotia1998** [comment]
@geekyd done please check!

**Comment by agbilotia1998** [comment]
@geekyd please run a travis build again here.

**Comment by mariobehling** [comment]
Please check again how sched.org is doing it. Please provide an updated screenshot and sample deployment to enable us to test this easily.

**Comment by agbilotia1998** [comment]
31016470-752caee8-a542-11e7-8158-7600c6111f70

**Comment by mariobehling** [comment]
Please use an arrow like in the image below and keep it to the font color. Don't use the track color. Use an svg arrow, please.

Also ensure there is some space to the left edge of the area.

screenshot from 2018-01-30 23-24-34

**Comment by agbilotia1998** [comment]
Okay @mariobehling .

**Comment by agbilotia1998** [comment]
screenshot from 2018-02-01 01-40-13
@mariobehling @geekyd done please review.

**Pull request merged**